src/core/src/bootstrap/EnvLayer.py

```diff
@@ -428,6 +429,27 @@ def write_with_retry_using_temp_file(file_path, data, mode='w'):
                     else:
                         raise Exception("Unable to write to {0} (retries exhausted). Error: {1}.".format(str(file_path), repr(error)))
 
+        @staticmethod
+        def delete_files_from_dir(dir_name, file_identifier_list, raise_if_delete_failed=False):
+            """ Clears all files from given dir. NOTE: Uses file_identifier_list to determine the content to delete """
+            for file_identifier in file_identifier_list:
+                files_to_delete = glob.glob(str(dir_name) + "/" + str(file_identifier))
+
+                for file_to_delete in files_to_delete:
+                    try:
+                        os.remove(file_to_delete)
+                    except Exception as error:
+                        error_message = "Unable to delete files from directory [Dir={0}][File={1}][Error={2}][RaiseIfDeleteFailed={3}].".format(
+                            str(dir_name),
+                            str(file_to_delete),
+                            repr(error),
+                            str(raise_if_delete_failed))
+
+                        if raise_if_delete_failed:
+                            raise Exception(error_message)
+                        else:
+                            print(error_message)
```

Comment:
nit: is print here a debug statement?

Reply:
We don't have a logger reference in EnvLayer so print is the way to log if needed